fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -164,6 +164,13 @@ private void incrementCallCount(FDBStoreTimer.Counts counter) {
         }
     }
 
+    private void incrementSize(FDBStoreTimer.Counts counter, int size) {
```

Comment:
Why have the more complicated contract that this checks whether counter is a size and does nothing? Why not just another overload of `increment` that takes an `amount`?

Reply:
My understanding is that we use the amount > 1 only for incrementing data size (although the underlying ```StoreTimer.increment(@Nonnull Count event, int amount)``` is not designed to be specific to data size). And we can see the method above has a specific name ```incrementCallCount```, which looks like specific for incrementing of calling count. If so, checker for ```counter.isSize()``` may be meaningful here. But I just looked at the similar code in ```InstrumentedReadTransaction``` and see it just simply have 2 overloads for ```increment```, one with amount argument and the other is to increment only 1. So I just changed the 2 methods here in same way. Please let me know if it looks good.